BUILD.gn

```diff
@@ -12,51 +14,29 @@ if (is_android || defined(is_docker)) {
 }
 
 config("external_config") {
+  visibility = [
+    ":*",
+  ]
   include_dirs = [ "include" ]
 }
 
 config("internal_config") {
-  include_dirs = [ "src" ]
-}
-
-group("bat-native-confirmations") {
-  public_deps = [
-    ":confirmations",
-  ]
-
-  data_deps = [
-    ":bat-native-confirmations-standalone",
-  ]
-}
-
-static_library("bat-native-confirmations-standalone") {
-  # TODO(bridiver) - only export symbols from `include`
-  complete_static_lib = true
-
-  output_dir = root_out_dir
-
-  configs -= [ "//build/config/compiler:thin_archive" ]
-
-  defines = [
-    "STANDALONE_BUILD",
-  ]
-
-  deps = [
-    ":confirmations",
+  visibility = [
+    ":*",
+    "//brave/test:*",
   ]
+  include_dirs = [ "src" ]
 }
 
-source_set("confirmations") {
+source_set("bat-native-confirmations") {
   public_configs = [ ":external_config" ]
   configs += [ ":internal_config" ]
 
-  output_name = "bat_native_confirmations"
-
-  if (is_mac || is_ios ) { 
-    libs = [
-      "Security.framework"
-    ]
-  }
+  visibility = [
+    ":*",
+    rebase_path("bat-native-ledger", dep_base) + ":*",
+    "//brave/test:*",
+  ]
 
   sources = [
     "src/bat/confirmations/notification_info.cc",
```

Comment:
header files need to be included here as well

Reply:
👍